src/components/RecentSnippets.jsx

```diff
@@ -8,24 +8,73 @@ import * as actions from '../actions';
 import '../styles/RecentSnippets.styl';
 
 class RecentSnippets extends React.Component {
+  constructor(props) {
+    super(props);
+    this.nextSetOfSnippets = this.nextSetOfSnippets.bind(this);
+    this.prevSetOfSnippets = this.prevSetOfSnippets.bind(this);
+  }
+
   componentDidMount() {
-    const { dispatch } = this.props;
-    dispatch(actions.fetchRecentSnippets);
+    const { dispatch, recent, pagination } = this.props;
+    let marker = null;
+
+    if (pagination.get('prev')) {
+      marker = recent.get(0) + 1;
+    }
+
+    dispatch(actions.fetchRecentSnippets(marker));
+  }
+
+  nextSetOfSnippets() {
```

Comment:
(just a thought) It took me a moment why `nextSetOfSnippets` uses the `prev` link :) So the idea behind naming in the API was that we return snippets in the "natural" order - newer first. I don't know, maybe we should stick to consistent naming everywhere. Or use terms "older" and "newer" instead to avoid the confusion. @ikalnytskyi , what do you think?

Reply:
Good point, I think `older` or `newer` will be more intuitive. Lets wait @ikalnytskyi

Reply:
@ikalnytskyi is ok with *Older* and *Newer*. 👍